components/AppBar/src/MDCAppBar.h

```diff
@@ -75,4 +75,30 @@
  */
 @property(nonatomic, strong, nonnull, readonly) MDCHeaderStackView *headerStackView;
 
+/**
+ Whether the App Bar should attempt to extract safe area insets from the view controller hierarchy
+ or not.
+
+ This behavior provides better support for App Bars on iPad, extensions, and anywhere else where the
+ view controller might not be directly behind the status bar / device safe area insets.
+
+ Enabling this behavior will do the following:
+
+ - Enable the same-named behavior on the headerViewController's headerView.
```

Comment:
Enable the same-named behavior on the headerViewController?

Reply:
Done.